src/client/app/shared/navbar/navbar.component.scss

```diff
@@ -0,0 +1,37 @@
+:host {
+    nav {
+        background-color: #1976D2;
+        a:hover {
+            box-shadow: 4px 3px 8px 2px #26588b;
+        }
+        img {
+            width: 96px;
+        }
+        box-shadow: 0 3px 5px -1px rgba(0, 0, 0, .2),
+        0 6px 10px 0 rgba(0, 0, 0, .14),
+        0 1px 18px 0 rgba(0, 0, 0, .12);
+        .active {
+            box-shadow: 4px 3px 8px 2px #26588b;
+        }
+    }
+    .app-navbar {
```

Comment:
Can you move this content into the `nav` element and get rid of the class `.app-navbar`?

Reply:
Hey Sorry, Just saw your message. Yes we can move the content of .app-navbar into nav